### PR TITLE
New package: gif.ski version 1.14.4

### PR DESCRIPTION
**Pull request by antonioalopezfernandez**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/151481)

### CODE REVIEW

**Comment by stephengillie** [comment]
Verify: 
image

**Comment by stephengillie** [comment]
This package installs and launches normally on a Windows 10 VM.

File: manifests/g/gif/ski/1.14.4/gif.ski.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: gif.ski
```
Comment (stephengillie) [inline manifests/g/gif/ski/1.14.4/gif.ski.installer.yaml:4]:
```suggestion
PackageIdentifier: ImageOptim.gifski
```
Please update file name and path to match this change.

File: manifests/g/gif/ski/1.14.4/gif.ski.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: gif.ski
```
Comment (stephengillie) [inline manifests/g/gif/ski/1.14.4/gif.ski.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: ImageOptim.gifski
```
Please update file name and path to match this change.

File: manifests/g/gif/ski/1.14.4/gif.ski.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: gif.ski
```
Comment (stephengillie) [inline manifests/g/gif/ski/1.14.4/gif.ski.yaml:4]:
```suggestion
PackageIdentifier: ImageOptim.gifski
```
Please update file name and path to match this change.

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @antonioalopezfernandez,

Please consider this updated `PackageIdentifier`, which generally should be formatted as `Publisher`.`PackageName` or similar. Otherwise, this PR appears ready to merge.

**Comment by stephengillie** [comment]
Automatic Validation ended with: 

>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. 
actual 'manifests\g\gif\ski\1.14.4', 
expected 'manifests\i\ImageOptim\gifski\1.14.4'

Please update file paths to match this `PackageIdentifier` change.

**Comment by antonioalopezfernandez** [comment]
> Automatic Validation ended with:
> 
> > Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest.
> > actual 'manifests\g\gif\ski\1.14.4',
> > expected 'manifests\i\ImageOptim\gifski\1.14.4'
> 
> Please update file paths to match this `PackageIdentifier` change.

Done, sorry for the inconvenience

**Review by stephengillie — APPROVED** [review]

**Pull request merged**